src/ServiceControl.Monitoring.Data/ISender.cs

```diff
@@ -2,7 +2,7 @@
 {
     using System.Threading.Tasks;
 
-    interface ISender
+    public interface ISender
```

Comment:
If this is a single method interface, it could just be a delegate `Func<byte[], Task>`